tests/util.py

```
@@ -2,14 +2,24 @@
 import time
 import binascii
 import os 
-
+import tempfile
+import shutil
 import mock
 
 
 DEFAULT_TIMESTAMP = datetime.datetime(2016, 1, 1)
 DEFAULT_ISO_TIME = time.mktime(DEFAULT_TIMESTAMP.timetuple())
 
 
+def mk_db_and_blob_dir():
+    db_dir = tempfile.mkdtemp()
+    blob_dir = tempfile.mkdtemp()
+    return db_dir, blob_dir
+
+def rm_db_and_blob_dir(db_dir, blob_dir):
+    shutil.rmtree(db_dir, ignore_errors=True)
```

Why ignore errors? What errors get ignored?